docs/source/app_developers_guide/docker_test_network.rst

```diff
@@ -212,59 +303,87 @@ setting.
-      [22:18:33.219 [MainThread] handler INFO] Setting setting sawtooth.validator.transaction_families changed from None to [{"family": "intkey", "version": "1.0"}, {"family":"sawtooth_settings", "version":"1.0"}, {"family":"xo", "version":"1.0"}, {"family":"sawtooth_validator_registry", "version":"1.0"}]
+      [22:18:33.219 [MainThread] handler INFO] Setting setting sawtooth.validator.transaction_families changed from None to [{"family": "intkey", "version": "1.0"}, {"family":"sawtooth_settings", "version":"1.0"}, {"family":"xo", "version":"1.0"}, ...
 
 #. Run the following command to check the setting change. You can use any
    container for this step. Also, you can specify any REST API on the network;
-   this example uses the REST API on the first validator node.
+   this example uses the REST API on the first node.
 
    .. code-block:: console
 
-      # sawtooth settings list --url http://sawtooth-rest-api-default-0:8008
+      root@0e0fdc1ab# sawtooth settings list --url http://sawtooth-rest-api-default-0:8008
 
    The output should be similar to this example:
 
-   .. code-block:: console
+   * For PBFT:
+
+     .. code-block:: console
+
+        sawtooth.consensus.algorithm.name: pbft
+        sawtooth.consensus.algorithm.version: 0.1
+        sawtooth.consensus.pbft.members=["0242fcde86373d0aa376055fc6...
+        sawtooth.publisher.max_batches_per_block=1200
+        sawtooth.settings.vote.authorized_keys: 0242fcde86373d0aa376...
+        sawtooth.validator.transaction_families: [{"family": "intkey...
 
-      sawtooth.consensus.algorithm.name: PoET
-      sawtooth.consensus.algorithm.version: 0.1
-      sawtooth.poet.initial_wait_time: 15
-      sawtooth.poet.key_block_claim_limit: 100000
-      sawtooth.poet.report_public_key_pem: -----BEGIN PUBL...
-      sawtooth.poet.target_wait_time: 15
-      sawtooth.poet.valid_enclave_basenames: b785c58b77152cb...
-      sawtooth.poet.valid_enclave_measurements: c99f21955e38dbb...
-      sawtooth.poet.ztest_minimum_win_count: 100000
-      sawtooth.publisher.max_batches_per_block: 200
-      sawtooth.settings.vote.authorized_keys: 03e27504580fa15...
-      sawtooth.validator.transaction_families: [{"family": "in...
+   * For PoET:
 
+     .. code-block:: console
+
+        sawtooth.consensus.algorithm.name: PoET
+        sawtooth.consensus.algorithm.version: 0.1
+        sawtooth.poet.initial_wait_time: 15
+        sawtooth.poet.report_public_key_pem: -----BEGIN PUBLIC KEY-----
+        MIIBIjANBgkqhki...
+        sawtooth.poet.target_wait_time: 5
+        sawtooth.poet.valid_enclave_basenames: b785c58b77152cbe7fd55ee3...
+        sawtooth.poet.valid_enclave_measurements: c99f21955e38dbb03d2ca...
+        sawtooth.publisher.max_batches_per_block: 100
+        sawtooth.settings.vote.authorized_keys: 036631291bbe87c3c9dde22...
+        sawtooth.validator.transaction_families: [{"family": "intkey", ...
+
+     ??? REVIEWERS: WHAT ABOUT ZTEST? `sawtooth.poet.ztest_minimum_win_count: 100000`
```

Comment:
ztest is not set in the docker-compose file used in this procedure.

Reply:
That's my question: Why not?  It used to be set in a previous version of the file.  Was it removed on purpose?  Will the default z-test setting cause any problems if the network runs for a while?

Reply:
Setting the ztest to a high number is a hack to avoid a situation where the nodes get deadlocked because they all fail the ztest at the same time. However, this feels like a weird place to offer that information. Maybe it should be in a note or tip?

Reply:
(i do not know if we removed it on purpose or not)

Reply:
What is the default ztest value?  (I can't find it in the current docs or the FAQ.)  Perhaps it defaults to 1000000, and that's why the setting no longer appears in `sawtooth settings list`.

Reply:
Talked to @peterschwarz, who says that the default ztest value is fine for an app-dev test network. I shall remove my question from the doc and leave the example output as it is.

Reply:
Removed question; marking this conversation as resolved.